raiden/api/rest.py

```diff
@@ -304,83 +305,129 @@ def restapi_setup_type_converters(
         flask_app.url_map.converters[key] = value
 
 
-class APIServer(Runnable):  # pragma: no unittest
-    """
-    Runs the API-server that routes the endpoint to the resources.
-    The API is wrapped in multiple layers, and the Server should be invoked this way::
+@dataclass
+class WebUIConfig:
```

Comment:
Using a config class prevents setting the flag to enable the webui without the correct config.